src/tools/android-studio.md

```diff
@@ -34,10 +34,21 @@ You can create a new project in one of several ways.
 
 ### Creating a new project
 
-To create a new Flutter project from the Flutter starter app template:
+The steps to create a new Flutter project from the Flutter starter app template are 
+slightly different for Android Studio and IntelliJ.
```

Comment:
**nit:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).

```suggestion
Creating a new Flutter project from the Flutter starter app template
differs between Android Studio and IntelliJ.
```